lib/backend/model/block.go

```diff
@@ -90,13 +90,22 @@ type AllocationBlock struct {
 	Allocations    []*int                `json:"allocations"`
 	Unallocated    []int                 `json:"unallocated"`
 	Attributes     []AllocationAttribute `json:"attributes"`
+	Deleting       bool                  `json:"deleting"`
```

Comment:
Sorry for the late feedback.  Is it worth trying to sync the different approaches to this flag across the different resources?  I.e. here we use a flag called Deleting, but the Affinity uses a State field?